clases/person.rb

```diff
@@ -1,15 +1,16 @@
 require_relative 'corrector'
 
 class Person
-  attr_reader :id
   attr_accessor :name, :age
+  attr_reader :id, :parent_permission, :rental
```

Comment:
- [ ] Kindly include `rental` in the setter method

Reply:
Well solved